segmentstore/server/src/main/java/io/pravega/segmentstore/server/DebugSegmentContainer.java

```diff
@@ -25,4 +27,13 @@
      * completed. If the operation failed, the future will be failed with the causing exception.
      */
     CompletableFuture<Void> registerSegment(String streamSegmentName, long length, boolean isSealed);
+
+    /**
+     * Creates a segment with target segment name and copies the contents of the source segment to the target segment.
+     * @param storage                   A storage instance to create the segment.
+     * @param sourceSegment             The name of the source segment to copy the contents from.
+     * @param targetSegment             The name of the segment to write the contents to.
+     * @throws Exception                In case of an exception occurred while execution.
+     */
+    void copySegment(Storage storage, String sourceSegment, String targetSegment) throws Exception;
```

Comment:
Have we thought whether it would be better to put `copySegment` into the `Storage` interface? That way we could take advantage of LTS-specific copy features (so that we do not need to read everything and write it back). @sachin-j-joshi what do you think?

Reply:
Ideally we want rename method implemented in Storage. But that also means implementing  it for RollingStorage with all its corner cases.
Given that the rename functionality is needed only in DR tool and only once for a container metadata segment. I suggested this short term work around of copying that segment.
In near future we'll implement rename in Storage. But for now I think this is good enough to unblock us.